csp/checks.py

```diff
@@ -45,9 +47,9 @@
 ]
 
 
-def migrate_settings() -> Tuple[Dict[str, Any], bool]:
+def migrate_settings() -> tuple[dict[str, Any], bool]:
```

Comment:
It's interesting that this is allowed with Python 3.8 with `from __future__ import annotations`